getid3/module.audio.mp3.php

```diff
@@ -1893,8 +1894,8 @@ public static function ClosestStandardMP3Bitrate($bit_rate) {
 
 	public static function XingVBRidOffset($version, $channelmode) {
 		static $XingVBRidOffsetCache = array();
-		if (empty($XingVBRidOffset)) {
-			$XingVBRidOffset = array (
+		if (empty($XingVBRidOffsetCache)) {
+            $XingVBRidOffsetCache = array (
```

Comment:
`$XingVBRidOffsetCache` was declared as static, but `$XingVBRidOffset` was used instead of it